homeassistant/components/device_tracker/automatic.py

```diff
@@ -131,27 +128,8 @@ def initialize_data(session):
                 refresh_token)
             yield from initialize_data(session)
             return True
-        except aioautomatic.exceptions.BadRequestError as err:
```

Comment:
I've also just changed this to more permissively fall back to launching configurator.